services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/system_monitor/_disk_usage.py

```diff
@@ -17,41 +21,177 @@
 from ..mounted_fs import MountedVolumes
 from ..notifications import publish_disk_usage
 
+_NODE_FILE_SYSTEM_PATH: Final[Path] = Path("/")
+
+
 _logger = logging.getLogger(__name__)
 
 
+class MountPathCategory(StrAutoEnum):
+    HOST = auto()
+    STATES_VOLUMES = auto()
+    INPUTS_VOLUMES = auto()
+    OUTPUTS_VOLUMES = auto()
+
+
+_SUPPORTED_ITEMS: Final[set[str]] = {
+    MountPathCategory.HOST,
+    MountPathCategory.STATES_VOLUMES,
+}
+
+
 async def get_usage(path: Path) -> DiskUsage:
     usage = await asyncio.get_event_loop().run_in_executor(
         None, psutil.disk_usage, f"{path}"
     )
     return DiskUsage.from_ps_util_disk_usage(usage)
 
 
+def get_relative_path(path: Path, dy_volumes_mount_dir: Path) -> Path:
+    try:
+        return path.relative_to(dy_volumes_mount_dir)
+    except ValueError:
+        return path
+
+
+def _get_normalized_folder_name(path: Path) -> str:
+    return f"{path}".replace("/", "_")
+
+
+def _have_common_entries(a: set[str], b: set[str]) -> bool:
+    return len(a & b) > 0
```

Comment:
MINOR: Since you are wrapping it, i think `return bool(a.intersection(b))` reads better.